pkg/types/clusterconfig/cluster_config_aws.go

```diff
@@ -326,6 +328,20 @@ var CoreConfigStructFieldValidations = []*cr.StructFieldValidation{
 			Validator: validateImageVersion,
 		},
 	},
+	{
+		StructField: "ImagePrometheusDCGMExporter",
+		StringValidation: &cr.StringValidation{
+			Default:   "quay.io/cortexlabs/prometheus-dcgm-exporter:" + consts.CortexVersion,
+			Validator: validateImageVersion,
+		},
+	},
+	{
+		StructField: "ImagePrometheusKubeStateMetricsExporter",
```

Comment:
nit: change name to `ImagePrometheusKubeStateMetrics`. The exporter suffix makes it too big and it's not part of the original name anyway. This change should propagate to the rest of the code